pkg/packagekit/updates.jsx

```diff
@@ -574,6 +698,55 @@ class OsUpdates extends React.Component {
                             });
                 })
                 .fail(this.handleLoadError);
+
+        this.getUptime();
+        this.callTracer();
+    }
+
+    callTracer() {
+        console.log("Spawn");
+        cockpit.spawn(["tracer", "--root"], { superuser: "try", pty: true })
```

Comment:
Would it do any good to run this as user? I suppose not, so use superuser: "require" instead.

This also needs a comment why this needs a PTY.